website/source/api/secret/ad/index.html.md

```diff
@@ -46,6 +46,14 @@ text that fulfills those requirements. `{{PASSWORD}}` must appear exactly once a
 * `userdn` (string, optional) - Base DN under which to perform user search. Example: `ou=Users,dc=example,dc=com`
 * `upndomain` (string, optional) - userPrincipalDomain used to construct the UPN string for the authenticating user. The constructed UPN will appear as `[username]@UPNDomain`. Example: `example.com`, which will cause vault to bind as `username@example.com`.
 
+### Other parameters
+
+* `last_rotation_tolerance` (string, optional) - Active Directory often shows a "pwdLastSet" time after Vault's because it takes
+a while for password updates to be propagated across a large cluster. By default, if Active Directory's last rotation time is
+within 5 seconds of Vault's, Vault considers itself to have been the last entity that rotated the password. However, if it's been 
+more than 5 seconds, Vault thinks that something rotated the password out-of-band, and re-rotates it so it will "know" it and be
+able to continue returning it. This may be too high for larger Active Directory clusters, and too low for smaller ones.
```

Comment:
Should this be the other way around? I figured a small cluster could have a smaller value here.

Reply:
Ha yes! Good eye. Guess I didn't have enough :coffee: this morning.